### PR TITLE
add ci

### PR DESCRIPTION
**Pull request by charlierudolph**
Added the credentials to AWS and I'll test the deployment process on this branch once we destroy the old one

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    name: install exosphere
+    command: |
+      curl --output exo --location --fail https://github.com/Originate/exosphere/releases/download/v0.35.0/exo-linux-amd64
+      chmod +x exo
+      sudo mv exo /usr/local/bin/
+
+version: 2
+jobs:
+  test:
+    working_directory: ~/edx
```
Comment (hugoesthere) [inline .circleci/config.yml]: `~/exosphere`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    docker:
+      - image: circleci/buildpack-deps
+    steps:
+      - checkout
+      - setup_remote_docker
+      - *installExosphereStep
+      - run:
+          name: run tests
+          command: exo test
+  deploy:
```
Comment (hugoesthere) [inline .circleci/config.yml:23]: Do we need to deploy on CI?

Comment (charlierudolph) [reply .circleci/config.yml:23]: I think it will be a nice test as it resembles a real project.

File: .circleci/config.yml (hunk 1 above)
Comment (charlierudolph) [reply .circleci/config.yml]: I'll go with `~/space-tweet`. The name of the folder actually doesn't matter

**Review by hugoesthere — APPROVED** [review]

**Pull request merged**